CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# [3.0.0-beta.0](https://github.com/SebastianM/angular-google-maps/compare/1.1.0...3.0.0-beta.0) (2020-07-20)
```

Comment:
```suggestion
# [3.0.0-beta.1](https://github.com/SebastianM/angular-google-maps/compare/1.1.0...3.0.0-beta.1) (2020-07-20)
```

Reply:
no, because changelog is for what we already released